source/BannerlordCoopInstaller/Installer.cs

```diff
@@ -0,0 +1,264 @@
+﻿using Microsoft.Win32;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Reflection;
+using System.Resources;
+using System.Text;
+using System.Windows.Forms;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+
+namespace BannerlordCoopInstaller
+{
+    public partial class BannerlordCoopInstallerForm : Form
+    {
+
```

Comment:
Tons of whitespace in this class. ctrl+K+F should clean it up, but lets trim this down